01_stereoplots/SWS_Analysis_BASICS_stereoplot.m

```diff
@@ -138,12 +146,21 @@ function SWS_Analysis_BASICS_stereoplot(colmap)
     end
     
     % search for input cmap
-    if idx==1 && ~isempty(which([colmap '.mat']))
-            loadmap=load([colmap '.mat']);
-            getfname=fieldnames(loadmap);
-            usecmap=loadmap.(getfname{1});        
-    elseif idx==1 && isempty(which([colmap '.mat']))
-            warning('Scientific colour-maps by F. Crameri not found! Check your MATLAB search path!')
+    if idx==1 && ~isempty(which('CrameriColourMaps7.0.mat'))
+       % warning if colormap name contains upper-case letters
+       if strcmp(colmap,lower(colmap))==0
+            disp(' ')
+            warning(['v1.08: bug in >crameri.m<. '  ...
+                     'To fix it, uncomment line 97 of this function. ' ...
+                     'This line standardizes all colormap names to ' ...
+                     'lower-case. As MATLAB is a case-sensitive ' ...
+                     'programming language, colormap names containing ' ...
+                     'upper-case letters are not longer found in the ' ...
+                     'provided MATLAB structure.'])
+        end
+            usecmap=crameri(colmap, 181);
+    elseif idx==1 && isempty(which('CrameriColourMaps7.0.mat'))
+            warning('Scientific colour maps by F. Crameri not found! Check your MATLAB search path!')
             return

```

Comment:
Would remove the warning within the code, as this bug has been fixed in v1.09, but would leave a comment in the header of this function.

```suggestion
    if idx==1 && ~isempty(which('CrameriColourMaps7.0.mat'))
            usecmap=crameri(colmap, 181);
    elseif idx==1 && isempty(which('CrameriColourMaps7.0.mat'))
            warning('Scientific colour maps by F. Crameri not found! Check your MATLAB search path!')
            return
```